staging/vhost-device-console/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Fixed
+
+### Deprecated
+
+## [0.1.0]
```

Comment:
We haven't done the release yet, so for now I wouldn't put anything in the Changelo, just the Unreleased template.

Reply:
Let' keep v0.1.0 in the changelog, since we will do the release after merging this.
BTW can you fix the style following: https://github.com/rust-vmm/vhost-device/pull/695